.travis.yml

```diff
@@ -1,13 +1,16 @@
 language: ruby
 rvm:
-  - 2.3.4
-  - 2.4.1
-  - ruby-head
```

Comment:
`ruby-head` install `2.7-dev` currently and the version is incompatible activerecord 4.